modules/add-contextual-data/tests/test_context_info_db.c

```diff
@@ -224,39 +247,65 @@ Test(add_contextual_data, test_import_with_valid_csv)
             "The context_info_db_is_indexed reports False after successful import&load operations.");
   fclose(fp);
 
-  TestNVPair expected_nvpairs_selector1[2] =
+  TestNVPair expected_nvpairs_selector1[] =
```

Comment:
@kvch Sorry, I thought you asked me to change the names of my _own_ tests.

These three `TestNVPair`s are introduced in 141fcb0b769092d9fab04ce2b7cd0a1b164533c8. I don't think that these should be changed since they contain meaningless values and are used exactly to test `selector1`, `selector2` and `selector3`.

Nevertheless, feel free to make a suggestion.